data/templates/accel-ppp/config_ipv6_pool.j2

```diff
@@ -5,7 +5,7 @@ AdvAutonomousFlag=1
 {%   if client_ipv6_pool.prefix is defined and client_ipv6_pool.prefix is not none %}
 [ipv6-pool]
 {%     for prefix, options in client_ipv6_pool.prefix.items() %}
-{{ prefix }},{{ options.mask }}
+{{ prefix }},{{ '64 ' if options.mask is not defined else options.mask }}
```

Comment:
Why not specify `<defaultValue>64</defaultValue>` for mask in XML? The no special treatment is required here.

SSTP already uses `get_accel_dict()` which blends in the default values automatically in XML.

Reply:
I tried it, but it had incorrect behavior
It starts to see "mask" as "prefix"
For example when I use "defaultValue" 64 in XML
and such template:
```
{%     for prefix, options in client_ipv6_pool.prefix.items() %}
{{ prefix }},{{ options.mask }}
{%     endfor %}

```
It generates the next config (if the `mask` was not added in configuration):
`set vpn sstp client-ipv6-pool prefix 2001:db8::/48`
```
[ipv6-pool]
2001:db8::/48,
mask,
```
It is incorrect, I expect prefix /64 by default
Expected configuration:
```
[ipv6-pool]
2001:db8::/48,64
```
If I add mask in configuration:
`set vpn sstp client-ipv6-pool prefix 2001:db8::/48 mask 64`
it generates:
```
[ipv6-pool]
2001:db8::/48,64
mask,

```